web3-functions/pyth-oracle-w3f-priceIds/pythUtils.ts

```diff
@@ -0,0 +1,122 @@
+import { Octokit } from "octokit";
+import YAML from "yaml";
+import { EvmPriceServiceConnection, Price } from "@pythnetwork/pyth-evm-js";
+
+export interface PythConfigStorage {
+  timestamp: number;
+  pythConfig: PythConfig;
+}
+export interface PythConfig {
+  pythNetworkAddress: string;
+  priceServiceEndpoint: string;
+  configRefreshRateInSeconds: number;
+  validTimePeriodSeconds: number;
+  deviationThresholdBps: number;
+  priceIds: string[];
+}
+
+export const addLeading0x = (id: string): string => {
+  if (id.startsWith("0x")) {
+    return id;
+  }
+  return "0x" + id;
+};
+
+export interface Web3Storage {
+  get(key: string): Promise<string | undefined>;
+  set(key: string, value: string): Promise<void>;
+  delete(key: string): Promise<void>;
+}
+
+export const shouldFetchPythConfig = (
```

Comment:
nit: i think it's more common to write this as `export async function(args): Ret { }`